modules/viewing-product-usage.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * operating/use-system-health-dashboard.adoc
+:_content-type: PROCEDURE
+[id="viewing-product-usage_{context}"]
+= Viewing product usage data
+
+[role="_abstract"]
+{product-title-short} provides product usage data for the number of secured Kubernetes nodes and CPU units for secured clusters based on metrics collected from {product-title-short} sensors. This information can be useful in estimating {product-title-short} consumption data for reporting.
+
+For more information on how CPU units are defined in Kubernetes, see link:https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-cpu[CPU resource units].
+
+[NOTE]
+====
+{ocp} provides its own usage reports; this information is intended for use with self-managed Kubernetes systems.
+====
+
+This page provides the following data:
```

Comment:
Which page?

Reply:
I meant this page in the UI, but am rewriting to avoid awkwardness.